store/iavl/tree.go

```diff
@@ -31,6 +31,7 @@ type (
 		GetVersionedWithProof(key []byte, version int64) ([]byte, *iavl.RangeProof, error)
 		GetImmutable(version int64) (*iavl.ImmutableTree, error)
 		SetInitialVersion(version uint64)
+		AvailableVersions() []int
```

Comment:
this is API breaking so cannot backport.

We can backport this PR if we remove this line in the backport

Reply:
im fine with that